src/main/resources/templates/add.ftl

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<body>
```

Comment:
You need to add a lang attribute to html tag and head tag underneath  with 
```
  <meta charset="UTF-8">
  <title>name of your app</title>
```

details:
http://htmlbook.ru/html/attr/lang
http://htmlbook.ru/html/head
http://htmlbook.ru/html/meta